napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -12,13 +12,22 @@
     QLabel,
     QPushButton,
     QVBoxLayout,
+    QWidget,
 )
-from superqt import QLabeledDoubleSlider
+from superqt import QEnumComboBox, QLabeledDoubleSlider
```

Comment:
```suggestion
from enum import Enum, EnumMeta
from superqt import QEnumComboBox, QLabeledDoubleSlider
```

pre-commit will move on proper place